config.template.yaml

```diff
@@ -7,10 +7,11 @@ admin_password: null
 
 # can also be specified as the 'BASE_URL' environment variable
 #
-# this is the base url of the judging app, e.g. 'https://judging.example.com/'.
-# make sure to have a trailing slash. this must be set for emailed links to
-# work properly. if you are setting DISABLE_EMAIL, you can set this setting to
-# '_unused_' (or anything else).
+# this is the base url of the judging app, e.g. 'https://judging.event.com/'
```

Comment:
We used `example.com` for a reason: it's a [reserved name](https://tools.ietf.org/html/rfc2606). `event.com` is a real domain owned by some random party.

Reply:
Thank you for this comment, I am enlightened to know this fact that is new to me.